.travis-setup.sh

```diff
@@ -17,42 +17,49 @@
 #  along with this program; if not, write to the Free Software
 #  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
-glib_version=2.49.4
+set -e -x
+
+glib_version=2.46.2
```

Comment:
Ok, so if this is a double-check on `configure.ac`, we should auto-generate `.travis-setup.sh` from `.travis-setup.sh.in` using the value of `required_glib_version` to ensure we only define this version once.